docs/3.x.x/getting-started/quick-start.md

```diff
@@ -1,285 +1,281 @@
 
-**#3 —** Add three fields in this Content Type.
+1. Let's now add this second content type:
 
-  - A `string` field named `title`.
-  - A `text` field named `content` (tick the `Display as WYSIWYG` in the `Advanced Settings` tab).
-  - A `media` field named `cover`.
+   Click the **+ Add Content Type** menu item.
 
-![Content Type Builder - List fields in Post](../assets/getting-started_list_fields.png)
+   Now enter a name for this New Content Type (we will call this one **"category"**), and write a Description. We will write **"Restaurant Categories"** for the description.
 
-**#4 —** Save. That's it!
+   Then click the **Save** button.
 
+2. Now you are ready to add the content type fields. We will add a **String** field for the Name and a **Relation Field** for creating a **Many to Many** relation between the Category Content Type and Restaurant Content Type.
 
-::: note
-See the [CLI documentation](../cli/CLI.md#strapi-generateapi) for more information on how to do it the hacker way.
+   The **Name** Field
+
+   Click on the **"+ Add New Field"** button.
+
+   Next we will click on the **String** field.
+
+   You should be under the **BASE SETTINGS** tab, in the **Name** field, type **"Name"**. This will be the name of the category.
+
+   Now, click on the **ADVANCED SETTINGS** tab, check **Required field** and **Unique field**. This field is required for each category and is not optional. Also, we want each category to have a unique name, so we check the **Unique field** box.
+
+   And then click the **Continue** button.
+
+3. We are now ready to add the second field, the **Relation** field for the **Many to Many relation** setting between Categories and Restaurants.
+
+   The **Relation** Field
+
+   Click the **"+ Add New Field"** button.
+
+   From here, click the **Relation** field.
+
+   This will bring you to the **Add New Relation** screen. We will change two elements on this page for the **Relation** field.
+
+   The first element to change is **Permission (Users-Permissions)** to **Restaurant** (on the right).
+
+   The second element to change is to click the **Many to Many** icon (in the middle). It should now read, **"Categories has and belongs to many Restaurants"**.
+
+   Then click the **Continue** button.
+
+4. Lastly, we will save this Content Type. Click the **Save** button.
+
+5. Wait for Strapi to restart.
+
+Verify in the left menu under **CONTENT TYPES**, that you see **Categories**, **Restaurants** and **Users**.
+
+We are now ready to [manage and add data to the Content Types](#_4-manage-and-add-data-to-content-type) we just created.
+
+Next, click on the **Strapi** logo (top left corner) to return the main administration panel.
+
+::: warning NOTE
+See the [CLI documentation](../cli/CLI.md#strapi-generateapi) for more information on how to add Content Types the hacker way.
```

Comment:
See the [CLI documentation](../cli/CLI.md#strapi-generateapi) for more information on how to add Content Types using the Strapi CLI.

Reply:
This was completely removed from both the quick start and the tutorial. Seem not applicable for those reading these guides.